guacml/models/random_forest.py

```diff
@@ -56,3 +57,6 @@ def predict(self, x):
     def feature_importances(self, x):
         feat_scores = self.model.feature_importances_
         return pd.Series(list(feat_scores), index=list(x.columns))
+
+    def get_state(self):
```

Comment:
I wonder if we could move this into the `BaseModel` as the default implementation and only override it when needed. Like for XGBoost.